e2e/smoke_test/smokeTest.spec.js

```diff
@@ -1,139 +1,126 @@
-const { setup, screen, SettingDefault } = require('../../src/index');
+const { loop } = require('../../src/index');
 
 describe('smoke test', () => {
     beforeAll(async () => {
-        await setup.LaunchLoop();
+        await loop.Launch();
     });
     describe('home screen', () => {
         it('has Active Carbohydrates section', async () => {
-            await screen.home.OpenActiveCarbohydratesChart();
-            await screen.home.CloseChart();
+            await loop.screens.home.OpenActiveCarbohydratesChart();
+            await loop.screens.home.CloseChart();
         });
         it('has Active Insulin section', async () => {
-            await screen.home.OpenActiveInsulinChart();
-            await screen.home.CloseChart();
+            await loop.screens.home.OpenActiveInsulinChart();
+            await loop.screens.home.CloseChart();
         });
         it('has Insulin Delivery section', async () => {
-            await screen.home.OpenInsulinDeliveryChart();
-            await screen.home.CloseChart();
+            await loop.screens.home.OpenInsulinDeliveryChart();
+            await loop.screens.home.CloseChart();
         });
         it('has Glucose section', async () => {
-            await screen.home.OpenGlucoseChart();
-            await screen.home.CloseChart();
+            await loop.screens.home.OpenGlucoseChart();
+            await loop.screens.home.CloseChart();
         });
         it('has Loop icon', async () => {
-            await screen.home.ExpectLoopNotYetRun();
+            await loop.screens.home.ExpectLoopNotYetRun();
         });
         it('has Loop icon has alert when not setup', async () => {
-            await screen.home.ExpectLoopStatusGlucoseDataAlert();
+            await loop.screens.home.ExpectLoopStatusGlucoseDataAlert();
         });
     });
     describe('settings', () => {
         beforeAll(async () => {
-            await screen.settings.Open();
+            await loop.screens.settings.Open();
         });
         afterAll(async () => {
-            await screen.settings.Close();
+            await loop.screens.settings.Close();
         });
         describe('general', () => {
             it('can be go set to closed loop', async () => {
-                await screen.settings.SetClosedLoop();
+                await loop.screens.settings.SetClosedLoop();
             });
             it('can be go set to open loop', async () => {
-                await screen.settings.SetOpenLoop();
+                await loop.screens.settings.SetOpenLoop();
             });
             it('can issue a report', async () => {
-                await screen.settings.IssueReportLabel();
+                loop.screens.settings.IssueReportLabel();
```

Comment:
I was mistakenly just checking for the presence of the label rather than interacting with it and opening the section. I have updated the code and test 

I have played around with the awaits and they are required when interacting, they aren't required when selecting (i.e. getting a certain button element)